src/components/layout/navbar-profile-menu.tsx

```diff
@@ -99,20 +99,22 @@ export function NavbarProfileMenu() {
                     tabIndex={opened ? 0 : -1}
                     ref={accountMenuItemRef}
                 />
-                {/* TODO: restore once page is built*/}
-                {/* <NavLink
-                    label="Reviewer Key"
-                    leftSection={<Lock aria-hidden="true" />}
-                    onClick={() => {
-                        window.alert('404. Design under construction')
-                    }}
-                    c="white"
-                    className={styles.navLinkProfileHover}
-                    role="menuitem"
-                    aria-label="Reviewer Key" 
-                    tabIndex={opened ? 0 : -1}
-                    ref={reviewerKeyMenuItemRef}
-                /> */}
+
+                {auth.isReviewer && (
+                    <NavLink
+                        label="Reviewer Key"
+                        leftSection={<Lock aria-hidden="true" />}
+                        onClick={() => {
+                            router.push('/account/keys')
+                        }}
+                        c="white"
+                        className={styles.navLinkProfileHover}
+                        role="menuitem"
+                        aria-label="Reviewer Key"
+                        tabIndex={opened ? 0 : -1}
+                        ref={reviewerKeyMenuItemRef}
+                    />
+                )}
                 <NavLink
```

Comment:
can you extract this into it's own `<ReviewerKeyLink />` component which has a `if (!auth.isReviewer) return null` in it?  That will eliminate the large `{` `}` block